### ISSUE TITLE
download the latest  helm version from bintray

### ISSUE THREAD

**Issue by rimusz** [deis/helm#123]
would be nice to have a shell script (bash/curl) to download the latest version of `helm` from bintray

**Comment by michelleN** [comment]
+1

**Comment by michelleN** [comment]
@rimusz for the README right?

**Comment by rimusz** [comment]
@michelleN  yep and for my Kube-Solo App too :)

**Comment by rimusz** [comment]
got this short script for OS X binary
````
#!/bin/bash

# get lastest OS X helm version from bintray

bin_version=$(curl -I https://bintray.com/deis/helm-ci/helm/_latestVersion | grep "Location:" | sed -n 's/.*git-\([^ ]*\)view/\1/p' | tr -d '/' | tr -d '\r' )

curl -L "https://dl.bintray.com/deis/helm-ci/helm-git-$bin_version-darwin-amd64.zip" -o helm.zip
````
it should be very easily adopted to download linux ones too

**Comment by michelleN** [comment]
:+1:

**Comment by sgoings** [comment]
I do know that Bintray offers a nice API to grab the latest version:

https://bintray.com/docs/api/#_versions

Which could then be used to grab the actual files (rather than greppin' and shtuff)

**Comment by rimusz** [comment]
@sgoings I will leave that fun for you :)

**Comment by sgoings** [comment]
![no](http://memesvault.com/wp-content/uploads/Nooo-Meme-04.jpg)

**Comment by rimusz** [comment]
yes baby yes :)

**Comment by kgrvamsi** [comment]
@rimusz u just made things so simple man.... :)

**Comment by rimusz** [comment]
updated script version:
````
#!/bin/bash

# get lastest OS X helm version from bintray

bin_version=$(curl -I https://bintray.com/deis/helm-ci/helm/_latestVersion | grep "Location:" | sed -n 's%.*helm/%%;s%/view.*%%p' )

curl -L "https://dl.bintray.com/deis/helm-ci/helm-git-$bin_version-darwin-amd64.zip" -o helm.zip
````

**Comment by rimusz** [comment]
implemented by #191
